### ISSUE TITLE
Filter output field list

### ISSUE THREAD

**Issue by adrianshort** [adrianshort/uk_planning_scraper#20]
Allow users to specify which fields they do or don't want included in the output.

Add `only` and `except` to the `params` hash in `Authority#scrape`.

Both of these should be a comma-separated list of field names.

Using `only` and `except` params at the same time throws an error.

We might need to consider how this would interact with potential options for a deep or shallow scrape, eg an option like `documents: true` which scrapes the contents of documents pages.

**Comment by adrianshort** [comment]
For `Authority#scrape`:

`def scrape(params, options = {})`

Note:

- `params` is for _what_ to scrape (the search terms sent to the site and the output desired) 
- `options` is for _how_ to scrape (configuring the scraper's speed, user agent, etc).

**Comment by KeithP** [comment]
if the fields a user specifies to exclude amounts to a whole tab then we should omit scraping that tab.

**Comment by adrianshort** [comment]
True. And that's going to be a bunch of fun to code because different systems put their fields on different pages, so you'd need a data structure breaking down which systems, pages and fields correspond.
